env/common/templates/galaxy/files/xsede_report.py.j2

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env  python
+"""
+!! This file is templated by Ansible, manual changes will have no effect.
+
+Report jobs to XSEDE API. Run with CRON and in Galaxy venv.
+"""
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+import psycopg2
+import requests
+
+galaxy_root = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.insert(1, os.path.join(galaxy_root, 'lib'))
+
+import galaxy.config
+from galaxy.util.script import app_properties_from_args, populate_config_args
+
+DESTINATION_MAPPING = {
+    "stampede_" : "stampede2.tacc.xsede",
+    "bridges_" : "bridges-large.psc.xsede"
+}
+SERVER_NAMES = ["test", "main"]
+OFFSET_FILE_NAME = "xsede_retry_offset"
+
+# The following values are templated by Ansible
+OFFSET_FILE_PATH = "{{ galaxy_root }}/bin/" + OFFSET_FILE_NAME
```

Comment:
This dir isn't writable and shouldn't be used for storing state data. How about `{{ galaxy_root }}/var/xede_report`?